### PR TITLE
feat: improve Nutri-Score game

### LINKED ISSUE #63: Nutri-Score game
### What
- Nutri-Score game
- [ ] #62 
- [ ] Make the Nutri-Score game accessible when the DEV mode is enabled
- [ ] Add a counter of the total of Nutri-Scores to validate
- [ ] Add an explanation that you need to select all the valid Nutri-Score A and validate

### Part of 
- #37

### PR DESCRIPTION
**Pull request by alexfauquette**
Fix #63

- The first commit is about
	- Making the game accessible in dev mode ("nutriscore" button in the menu)
	- stick the validate/reject buttons at the  bottom of the page
	- allows modifying image size (out of the issue)
- The second commit is just the result of `yarn prettier`

image

### CODE REVIEW

File: src/pages/nutriscoreValidator/index.jsx [hunk 1]
```diff
@@ -56,27 +61,52 @@ export default function NutriscoreValidator() {
...
           <MenuItem value="b">Nutriscore B</MenuItem>
           <MenuItem value="c">Nutriscore C</MenuItem>
           <MenuItem value="d">Nutriscore D</MenuItem>
           <MenuItem value="e">Nutriscore E</MenuItem>
         </TextField>
-      </Box>
+        <Box sx={{ mx: 2, width: 500, maxWidth: 500, textAlign: "left" }}>
+          <Typography gutterBottom>Image sizes</Typography>
+          <Slider
+            aria-label="Temperature"
```
Comment (nobeeakon) [inline src/pages/nutriscoreValidator/index.jsx:80]: :v

File: src/components/ResponsiveAppBar.jsx [hunk 2]
```diff
@@ -44,9 +45,7 @@ const ResponsiveAppBar = () => {
 
   const { devMode: isDevMode } = React.useContext(DevModeContext);
```
Comment (nobeeakon) [inline src/components/ResponsiveAppBar.jsx:46]: haha made me think we could rename devMode

File: src/pages/nutriscoreValidator/index.jsx (hunk 1 above)
Comment (alexfauquette) [inline src/pages/nutriscoreValidator/index.jsx:80]:
```suggestion
            aria-label="Image size"
```

**Review by nobeeakon — APPROVED** [review]

**Comment by teolemon** [comment]
Explaining what you need to do ("Select all the products that have/don't have…") would be helpful. I didn't immediately understand how to play the game

**Pull request merged**